src/simuq/braket/braket_provider.py

```diff
@@ -87,13 +91,54 @@ def compile(
             self.prog = self.ahs_prog
             self.layout = layout
 
+        elif self.provider == "ionq":
+            if device == "Harmony":
+                nsite = 11
+            elif device == "Aria-1":
+                nsite = 23
+            elif device == "Aria-2":
+                nsite = 23
```

Comment:
Should this come from device properties instead?

Reply:
May not be a bad idea. My original thought was to copy what's being done with IonQ direct access so we wouldn't need an AWS account to do this; but I suppose that's what setting device to an int can do (see below)